django/cantusdb_project/main_app/views/chant.py

```diff
@@ -708,7 +708,19 @@ def get_context_data(self, **kwargs) -> dict:
             url_with_search_params = current_url + "?"
             
         context["url_with_search_params"] = url_with_search_params
-        print(self.request)
+
+        if search_keyword:
+            cantus_index_url = "http://cantusindex.org/search?t={k}".format(k=search_keyword)
```

Comment:
It could be I'm not understanding the purpose of the changes here, but I'm wondering why you are building html strings here? Could they not be included in an appropriate template?

Reply:
This is a good point – for some reason, I was worried about applying the right formatting (i.e. bolding of the search keyword) in the template, so I decided to build the string in the view. But now that I think a little more about it, we have everything we need to do this all in the template. Let me make this change.

Reply:
done! thank you for this suggestion!

Reply:
cool, I think this is cleaner and more logical